mydependency/index.html

```diff
@@ -10,97 +10,98 @@
   <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&amp;display=swap" rel="stylesheet" type="text/css"/>
   <!-- JQuery -->
   <script src="https://code.jquery.com/jquery-latest.min.js"></script>
-  <!-- Our JS -->
-  <script type="text/javascript" src="../pypi_checker.js"></script>
+  <!-- Marked parser -->
+  <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
   <!-- Favicon -->
   <link href="https://gist.githubusercontent.com/astariul/c09af596e802e945d3032774e10e1047/raw/f693a2e2b65966494da082887bc4be2917f615e4/random_icon.svg" rel="icon"/>
-  <style>
-   body { font-family: "Montserrat"; }
-   code { 
-    font-size: 100%;
-    display: inline-block !important;
-    font-size: 1.4rem; }
-   .header {
-    margin-top: 15rem;
-    font-size: 3.6rem;
-    font-weight: 400;
-    letter-spacing: -.1rem; }
-   .text-header {
-	 text-transform: uppercase;
-	 font-size: 1.4rem;
-	 letter-spacing: .2rem;
-	 font-weight: 600; }
-   .version {
-	 font-size: 2.3rem; 
-	 font-style: italic;
-    font-weight: 300; }
-   .elem { margin: 1rem 0rem; }
-   .danger-button {
-    text-transform: none;
-    border-color: crimson;
-    font-family: "Montserrat";
-    color: crimson;
-    pointer-events: none;
-    font-size: 1.2rem;
-    margin-bottom: .2rem;
-   }
-   button:hover {
-    border-color: darkcyan;
-    color: darkcyan;
-   }
-  </style>
+  <!-- Custom Styles -->
+  <link href="../static/package_styles.css" rel="stylesheet" />
+  <!-- Our JS -->
+  <script type="text/javascript" src="../static/pypi_checker.js"></script>
+  <script src="../static/package_page.js" defer></script>
   <title>
    Pypi - YourCompany
   </title>
  </head>
-  <body>
-   <div class="container">
+ 
+<body>
+  <div class="container">
     <section class="header">
-     mydependency
-     <span>
-     </span>
-     <span id='latest-version' class="version">
-      1.0
-     </span>
+      <a href="/github-hosted-pypi" class="goback-button">
```

Comment:
Can we use a relative path here ?
Because if `/github-hosted-pypi` is hardcoded, it means people who use the repository as a template will have to modify it manually (not working outside of the box)

Reply:
yeah is better to leave that as a absolute path, kinda like this `<a href="./index.html">`